README.md

```diff
@@ -1,11 +1,10 @@
-# lambda-template
-A GitHub template for quickly starting a new AWS lambda project.
+# lambda-ebs-cleanup
+An AWS lambda to scan all regions for unattached EBS volumes and delete them.
 
-## Naming
-Naming conventions:
-* for a vanilla Lambda: `lambda-<context>`
-* for a Cloudformation Transform macro: `cfn-macro-<context>`
-* for a Cloudformation Custom Resource: `cfn-cr-<context>`
+This lambda will scan all regions active for the current account, looking
+for EBS volumes older than a specified age (default five minutes) that have
+a status of either `error` or `available` (aka 'unattached'). Once all
+matching volumes are found, deletes are issued for each.
 
```

Comment:
Do we want to add to the readme about the how to add a tag to the volume so that the unattached EBS volume doesn't get deleted?

Reply:
good idea, I'll add that